collections/ordicat/meta.json

```diff
@@ -0,0 +1,10 @@
+{
+  "description": "16 16 pixel little cats",
+  "discord_link": "",
+  "icon": "https://turbo.ordinalswallet.com/inscription/preview/780de371909574be2f19aced7dcb62a32bcedca5cb7980aed46b683b8263d4a4i0",
+  "inscription_icon": "780de371909574be2f19aced7dcb62a32bcedca5cb7980aed46b683b8263d4a4i0",
+  "name": "OrdiCat",
+  "slug": "ordicat",
+  "twitter_link": "https://twitter.com/ordicat_btc",
+  "website_link": "https://ordicat.fun/"
+}
```

Comment:
Jani Jarrett's wallet and mark lines 305-310 I don't know what I was doing on 69 but it just flowed kind of. If it's wrong please let me fix I'm still learning and need some directions please.

Reply:
You can see it in detail here
https://help.magiceden.io/en/articles/7957891-guide-to-listing-bitcoin-ordinal-inscriptions-on-magic-eden

This site is a tool
We made this site ourselves
https://ordinalshelper.github.io/
You can see it by uploading the inscriptions json from the Inscriptions Viewer.